package.json

```diff
@@ -77,6 +77,6 @@
     "rimraf": "^2.6.3",
     "ts-jest": "^23.10.4",
     "tslint": "^5.11.0",
-    "typescript": "~3.2.1"
+    "typescript": "^3.3.1"
```

Comment:
you should keep it as `~3.2.1`

Reply:
Yeah I was playing around because I noticed that for some reason the tests were typing the imported rules as `any`.

I thought maybe it was a weird bug in 3.2.1, so I upgraded, but it turned out it was because the `tests` folder wasn't in the `includes` in `tsconfig.json`.

I will revert this.